feat(native): clarifying crashpad behavior (#13689)
getsentry/sentry-docs: docs/platforms/native/advanced-usage/container-environments/index.mdx

---
title: Container Environments
description: "How to use the Sentry Native SDK in container environments."
sidebar_order: 2000
---

> ## Database-path on a mounted volume

Reviewer: ```suggestion
## Database Path on a Mounted Volume
```

> The Sentry Native SDK uses a [database path](https://docs.sentry.io/platforms/native/configuration/options/#database-path) to store events and crash reports. When you are using a containerized environment, you may want to mount a volume to persist the database across container restarts to avoid losing this data.

Author: Should we have some more detailed explanation here as to what the `database_path` is used for?

Reviewer 2: Like what, for example?

Author: As in, what kind of data the different backends store in the database path, and when these get picked up. As talked about, this might just warrant another advanced usage docs instead of me shoehorning it in here.

## Waiting for `Crashpad` to Finish

> Since SDK version [0.8.3](https://github.com/getsentry/sentry-native/releases/tag/0.8.3) the [option `crashpad_wait_for_upload`](https://docs.sentry.io/platforms/native/configuration/options/#crashpad-wait-for-upload) enables waiting for the `crashpad_handler` (on Linux) to finish its work before a shutdown-after-crash.

Reviewer: ```suggestion
Starting with SDK version [0.8.3](https://github.com/getsentry/sentry-native/releases/tag/0.8.3), the [option `crashpad_wait_for_upload`](https://docs.sentry.io/platforms/native/configuration/options/#crashpad-wait-for-upload) option allows the application (on Linux) to wait for the `crashpad_handler` to finish before a shutdown-after-crash.
```

> In SDK versions before 0.8.3, one could use a script similar to the example below to tie container shutdown to the `crashpad_handler` process.

Reviewer: ```suggestion
In SDK versions older than 0.8.3, you could use a script similar to the example below to tie container shutdown to the `crashpad_handler` process:
```
